fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportJobListener.java

```diff
@@ -44,7 +49,7 @@ public void afterJob() {
             if (jobExecution.getEndTime() != null) {
                 totalJobExecutionMilliSeconds += (jobExecution.getEndTime().getTime() - jobExecution.getStartTime().getTime());
             } else {
-                totalJobExecutionMilliSeconds += (currentExecutionEndTimeInMS - jobExecution.getStartTime().getTime());
```

Comment:
Not if you want Milliseconds, you can just pass the unit into java.time 
`TimeUnit.NANOSECONDS.toMillis(VALUE);`

Reply:
nice, I like this!!

Reply:
done